tar/tarsnap.1-mdoc.in

```diff
@@ -883,6 +883,12 @@ Note that due to network buffering the file position will not align precisely
 with how much data has been sent to or received from the
 .Nm
 server.
+.It SIGSTOP & SIGSTP
+On receipt of a SIGSTOP or SIGSTP signal,
+.Nm
+will suspend the current process.  Upon receiving a SIGCONT,
```

Comment:
This wording seems a bit off.  *tarsnap* isn't suspending anything; it is *being suspended*.  SIGSTOP is handled by the kernel without even entering userland.

I agree that it's worth mentioning that tarsnap will reconnect to the server and resume operations, but I don't know that list is the right place to mention it.  Maybe a separate paragraph after the list?

Reply:
Good point about the kernel suspending it.

In an email on 2019-08-12 ("Document ^Z"), you suggested adding to the list, and I think that's better for visibility than adding a paragraph after the list.  In particular, you wrote:
> A few people recently have asked me if tarsnap can handle being suspended.      
> The answer is "yes, of course" but it's clearly something which needs to be     
> better documented.                                                              
>
> Can you change the "SIGNALS" section of the tarsnap manpage from "provides  
> special treatment of the following signals" to "handles the following       
> signals" and add "SIGSTOP & SIGTSTP" with some verbiage about how these     
> signals suspend the tarsnap process, but tarsnap will reconnect to the         
> server when it wakes up?                                                        
>
> This might also belong in the FAQ.  Basically I just want to dispel the idea    
> that suspending tarsnap will make its current archive fail.

I was going to add ^Z to the website.  For the man-page, how about about:

SIGSTOP & SIGSTP

On receipt of a SIGSTOP or SIGSTP signal, the kernel will
suspend tarsnap.  Upon receiving a SIGCONT signal, tarsnap will
reconnect to the server and continue the specified task.

Reply:
Looks good.  Maybe "reconnect to the server if necessary" since being suspended for a few seconds won't cause connections to time out and dry runs don't connect at all.

Also remember to mdoc2man.sh after rebasing.